src/main/java/javax/measure/quantity/Time.java

```diff
@@ -34,6 +34,16 @@
 /**
  * Period of existence or persistence. The metric system unit for this quantity is "s" (second).
  *
+ * The second, symbol s, is the SI unit of time. It is defined by taking the fixed
+ * numerical value of the caesium frequency ∆νCs, the unperturbed ground-state
+ * hyperfine transition frequency of the caesium 133 atom, to be 9 192 631 770 when
+ * expressed in the unit Hz, which is equal to s^−1.
```

Comment:
If we allow Unicode character elsewhere (as in ∆ν), then we could as well rewrite s^-1 as s⁻¹.

Reply:
Good catch!